tests/test_content.py

```diff
@@ -107,7 +108,8 @@ def test_render_question(framework, question_id, question_set, question):
 
 
 def strip_paragraph_wrapper(value):
-    if value.startswith('<p>'):
-        value = value[3:-4]
+    match = re.fullmatch(r"^(<p.*?>)(.+?)(</p>)", value)
+    if match:
+        return match.group(2)
```

Comment:
Could you add a commit message explaining why this change is needed?

Reply:
:+1: 

    Modify test to be able to strip out more complex paragraph tags. We need to do this because the previous logic was only able to strip '<p>'. However, as of https://github.com/alphagov/digitalmarketplace-content-loader/pull/77, we get more complex paragraph tags such as '<p class="govuk-body">'. The new regex should be able to detect these more complex paragraph tags and allow us to strip them.